_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/tasks/certificates.yml

```diff
@@ -43,6 +43,8 @@
         creates: cert-CNCIAgent-localhost.pem
       - role: controller
         creates: cert-Controller-localhost.pem
+      - role: agent,netagent
+        creates: cert-CNAgent-NetworkingAgent-localhost.pem
```

Comment:
Does this mean you will always create a dual role cert, in case it might be needed?